src/util/user-home-dir.js

```diff
@@ -4,7 +4,8 @@ import ROOT_USER from './root-user.js';
 
 const path = require('path');
 
-const userHomeDir =
-  process.platform === 'linux' && ROOT_USER ? path.resolve('/usr/local/share') : require('os').homedir();
+export const home = require('os').homedir();
+
+const userHomeDir = process.platform === 'linux' && ROOT_USER ? path.resolve('/usr/local/share') : home;
```

Comment:
What about other operating systems like BSD? I wonder if we should use this root logic for those, too.